### PR TITLE
Add missing absolute imports

### PR DESCRIPTION
**Pull request by donatoaguirre24**
#### :page_facing_up: Description:

This PR adds all the missing absolute imports with exception of the Title since there's already an open PR to remove it.

---

#### :heavy_check_mark: Tasks:

* Add missing absolute imports
* Refactor the auth middleware since https://github.com/axios/axios/issues/2190 was fixed on `v0.20.0`
* Remove unused dependencies `node-sass` and `autoprefixer` (tailwind leftovers)
* Remove unnecessary `shallowEqual,` more info on when is needed [here](https://twitter.com/donatoaguirre24/status/1295804812077006849) 
* Update the `README` to reflect the new styling approach.

---

#### :play_or_pause_button: Demo:
* No UI changes were made

@loopstudio/react-devs

### CODE REVIEW

File: src/components/App/index.js [hunk 1]
```diff
@@ -1 +1 @@
-export { default } from './App';
```
Comment (lucaleivaloop) [inline src/components/App/index.js:1]: I'm unsure about using absolute imports when the imported is within the same folder, I'm curious, why are you choosing to do so?

Comment (donatoaguirre24) [reply src/components/App/index.js:1]: Consistency mainly but also it makes refactoring easy. Moving things around and copy pasting, becomes less error-prone

**Review by mwolman — APPROVED** [review]
🖤

**Review by Rminetti — APPROVED** [review]

File: src/services/middlewares/auth.js [hunk 2]
```diff
@@ -7,9 +7,6 @@ const authenticator = (httpClient, store) => {
     if (session) {
       Object.assign(config.headers, session);
     } else {
-      // Waiting for https://github.com/axios/axios/issues/2190
-      /* eslint-disable no-param-reassign */
-      config.params = config.params || {};
```
Comment (mconiglio) [inline src/services/middlewares/auth.js:12]: 🎉

**Review by mconiglio — APPROVED** [review]

File: src/components/App/index.js (hunk 1 above)
Comment (lucaleivaloop) [reply src/components/App/index.js:1]: For me is an easy way to see dependants and if you move the parent (or child) around you should either move them or rename the import. I'm still on the fence, I might bring this up on today's council, people seems to be siding with your take though. Good PR BTW, many relevant improvements.

File: src/services/middlewares/auth.js (hunk 2 above)
Comment (donatoaguirre24) [reply src/services/middlewares/auth.js:12]: I have been waiting for this day 😂

File: src/components/App/index.js (hunk 1 above)
Comment (donatoaguirre24) [reply src/components/App/index.js:1]: I understand your point and I think is mainly a matter of taste so both approaches are valid. Let me know if you want me to change something here 🤙🏻

**Review by Tiago-Zarzavidjian — APPROVED** [review]

**Review by lucaleivaloop — APPROVED** [review]

**Pull request merged**